starlite/app.py

```diff
@@ -158,7 +158,7 @@ class Starlite(Router):
 
     def __init__(
         self,
-        route_handlers: List["ControllerRouterHandler"],
+        route_handlers: Optional[List["ControllerRouterHandler"]] = None,
```

Comment:
I think you need to update your fork @wassafshahzad - the current signature for this is:

```py
route_handlers: "OptionalSequence[ControllerRouterHandler]" = None,
```